Sources/XCMetricsPlugins/GitPlugin.swift

```diff
@@ -0,0 +1,65 @@
+// Copyright (c) 2021 Spotify AB.
+//
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import Foundation
+import XCMetricsClient
+import XCMetricsUtils
+
+public struct GitPlugin {
+    
+    public enum GitData {
```

Comment:
I like this! 
Can you please format the comment with the documentation format so it shows up nicely in Xcode?
```
/// The current branch name.
case branch
```

Reply:
thanks! done